packages/@tailwindcss-upgrade/src/template/codemods/theme-to-var.ts

```diff
@@ -80,7 +80,7 @@ export function themeToVar(
   return rawCandidate
 }
 
-export function createConverter(designSystem: DesignSystem) {
+export function createConverter(designSystem: DesignSystem, { pretty = false } = {}) {
```

Comment:
The name `pretty` is not super clear here. The difference is that add spacing around the separator character right? Should it be `spacingAroundSeparator`? Not great either but at least you know what it's doing haha

Reply:
How about `prettyPrint` or `prettyFormat`?

`spacingAroundSeparator` is correct but such a long name 😅

Reply:
Sounds reasonable :)